### DISCUSSION TITLE
Can't use Night's Edge and Excalibur?

### DISCUSSION THREAD

**Discussion by Coosis** [Pryaxis/TShock#2906] [Q&A]
In my server, a friend of mine can't wield both the Night's Edge and Excalibur. Everytime he uses them, he gets webbed. After looking through the banned item list, I'm sure they were not banned. Maybe it had something to do with Power Glove? Sometimes when he takes it off it works.
Would love to know how to deal with this.

**Comment by sgkoishi** [comment]
Duplicate of #2804. Fixed for the next release, or download the experiment version.
